src/eva/core/data/datasets/embeddings/classification/multi_embeddings.py

```diff
@@ -77,6 +77,7 @@ def _load_embeddings(self, index: int) -> torch.Tensor:
 
         # Load embeddings and stack them accross the first dimension
         embeddings = [torch.load(path, map_location="cpu") for path in embedding_paths]
+        embeddings = [embedding.unsqueeze(0) if embedding.ndim==1 else embedding for embedding in embeddings]
```

Comment:
We now have 3 loops over embedding_paths / embeddings.

Could replace by something like this?

```python
embeddings = []
for path in embedding_paths:
  embedding = torch.load(os.path.join(self._root, path), map_location="cpu")
  embedding = embedding.unsqueeze(0) if embedding.ndim==1 else embedding
```